clusters/fixed-1.yaml

```diff
@@ -1,2 +1,8 @@
+overrides:
+  ceph-deploy:
+    conf:
+      global:
+        mon pg warn min per osd: 1
+        osd pool default size: 1
```

Comment:
These two override settings mean that there's no need to have three OSDs.  Better would be to set osd chooseleaf type 0, so that replication works as expected, along with perhaps osd pool default pg/pgp num so the pools get created large enough (don't know if the default is too small or not).  That would be testing more like a real configuration.